### PR TITLE
Singleproductpage

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Please review my component

### CODE REVIEW

**Review by HrishikeshMahalle — APPROVED** [review]
Couldn't spot any major issue. Good to Merge. Also Beautiful HomePage.

File: pages/singleproduct.html [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+      sizes="32x32"
+      href="../components/logo/android-chrome-192x192.png"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+      integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
```
Comment (surbhi84) [inline pages/singleproduct.html:22]: you can import fa icons in css.

File: pages/singleproduct.html [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+            src="../components/images/home-logo.png"
+            alt="logo"
+            class="logo-design"
+          />
+          <a href="#" class="logo">Trekkart Store</a>
+        </div>
+  
+        <nav class="navbar">
+          <a href="/" class="href">Home</a>
+          <a href="./products.html" class="href">Shop Now</a>
```
Comment (surbhi84) [inline pages/singleproduct.html:39]: classname can be improved.

**Review by surbhi84 — APPROVED** [review]
rest looks good to me

**Pull request merged**